sources/osgShader/Compiler.js

```diff
@@ -1058,6 +1078,8 @@ Compiler.prototype = {
 
         func._map = {};
```

Comment:
There is multiple time reset field like:
func._text = [];
func._mapTraverse = [];
this._traversedNodeList = {};

is it possible to write a function that reset it, I feel like it will grow and will be error prone